doc/cli/operator-sdk_bundle_create.md

```diff
@@ -27,6 +28,7 @@ $ tree ./deploy/olm-catalog/test-operator/
 	└── test-operator.v0.1.0.clusterserviceversion.yaml
 └── metadata
 	└── annotations.yaml
+```
 
```

Comment:
We need to update the website dir as well. Could you please check this file there and ensure that both are equals?